File: src/main/java/com/jessica/guess/game.java
package com.jessica.guess;

import java.util.Random;
import java.util.Scanner;

public class game {
    public static void main(String[] args) {
        Random random=new Random();
        int secret=random.nextInt(10)+1;
        Scanner scanner=new Scanner(System.in);
        int number=0;
        while(number !=secret) {
            System.out.println("Please enter a number:");
            number = scanner.nextInt();
            if (number < secret) {
                System.out.println("higher");
            }else if(number>secret){
                System.out.println("lower");
            }else{
                System.out.println("the number is"+secret);
            }
        }

    }
}

File: src/main/kotlin/com/Game.kt
package com

import java.util.*
import kotlin.collections.RandomAccess

fun main() {
    val secret =Random().nextInt(10)+1
//    val scanner=Scanner(System.`in`)
    var number =0
    while (number!=secret){
        println("plese enter number:")
        number = readLine()!!.toInt()
    if (number> secret){
        println("lower")
    }else if(number<secret){
        println("higher")
    }else{
        println("the number is$number")
    }
    }
}